include/pinocchio/autodiff/casadi-algo.hpp

```diff
@@ -133,6 +145,8 @@ namespace pinocchio
       ADFun ad_fun, ad_fun_derivs;
       ADFun fun, fun_derivs;
       std::vector<DMMatrix> fun_output, fun_output_derivs;
+      long fun_operation_count, fun_derivs_operation_count;
```

Comment:
```suggestion
      casadi_int fun_operation_count, fun_derivs_operation_count;
```

Casadi store int in casadi_int type (long long int). To avoid conversion issue, it's best to use the same type.

Reply:
Thank you Joris, this is now resolved.